internal/pkg/daemon/selinuxprofile/setup.go

```diff
@@ -38,14 +38,14 @@ func Setup(ctx context.Context, mgr ctrl.Manager, l logr.Logger) error {
 	if err != nil {
 		return errors.Wrap(err, "creating policy wrapper template")
```

Comment:
```suggestion
		return errors.Wrap(err, "creating profile wrapper template")
```
also renaming the above `policyWrapper` to `profileWrapper`?